plugins/org.jkiss.dbeaver.ext.gbase8s/plugin.xml

```diff
@@ -0,0 +1,285 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?eclipse version="3.2"?>
+<plugin>
+	<extension point="org.jkiss.dbeaver.generic.meta">
+		<meta id="gbase8s"
+			class="org.jkiss.dbeaver.ext.gbase8s.model.meta.GBase8sMetaModel"
+			driverClass="com.gbasedbt.jdbc.Driver" />
+	</extension>
+	<extension point="org.jkiss.dbeaver.dataSourceProvider">
+		<datasource
+			class="org.jkiss.dbeaver.ext.gbase8s.GBase8sDataSourceProvider"
+			id="gbase8s"
+			description="GBase 8s connector"
+			icon="icons/gbase8s_icon.png"
+			parent="generic"
+			dialect="gbase8s"
+			label="GBase 8s">
+			<tree path="generic" label="GBase 8s data source"
```

Comment:
I really don't like the idea to copy-paste entire metadata tree here. Besides other issues we will loose all localizations we already ave for generic drivers.
Can we use `treeInjection` instead of copy-paste? See examples in db2i, gaussdb, hive, etc

Reply:
Thank you for your feedback! I understand your concerns about copying and pasting the entire metadata tree, especially regarding the potential loss of localization for the generic drivers. In response, I’ve replaced the metadata tree with the `treeInjection` approach as suggested, referring to the examples from DB2i. The updates have been pushed.

Please let me know if you have any further suggestions or if you need anything else!

Reply:
Please remove now unused localization as well.

Reply:
I've already removed the unused localization strings as requested.